src/theme/brave-default.ts

```diff
@@ -16,8 +16,8 @@ const theme: ITheme = {
     brandBatInteracting: colors.blurple500,
     brandBatActive: colors.blurple200,
     // regular controls
-    defaultControl: colors.grey800,
-    defaultControlInteracting: colors.black,
+    defaultControl: colors.grey500,
```

Comment:
cc @rossmoody changes in the theme applies for all select boxes in our app.
same as above just checking this is what we want

Reply:
we can dismiss this change.